compass/ocean/tests/internal_wave/rpe_test/__init__.py

```diff
@@ -0,0 +1,51 @@
+from compass.testcase import TestCase
+from compass.ocean.tests.internal_wave.initial_state import InitialState
+from compass.ocean.tests.internal_wave.forward import Forward
+from compass.ocean.tests.internal_wave.rpe_test.analysis import Analysis
+from compass.ocean.tests import internal_wave
+
+
+class RpeTest(TestCase):
+    """
+    The reference potential energy (RPE) test case for the internal wave
+    test group performs a 20-day integration of the model forward in time at
+    5 different values of the viscosity at the given resolution.
+
+    Attributes
+    ----------
+    resolution : str
+        The resolution of the test case
+    """
+
+    def __init__(self, test_group):
+        """
+        Create the test case
+
+        Parameters
+        ----------
+        test_group : compass.ocean.tests.internal_wave.InternalWave
+            The test group that this test case belongs to
+        """
+        name = 'rpe_test'
+        super().__init__(test_group=test_group, name=name)
+
+        nus = [1, 5, 10, 20, 200]
```

Comment:
These are different from the viscosities in the legacy test case and in the original paper.  I think these might be copied from the baroclinic channel?

Following, https://github.com/MPAS-Dev/compass/blob/legacy/ocean/internal_waves/5km/rpe_test/plot.py#L10
```suggestion
        nus = [0.01, 1, 15, 150]
```
The formatting of the step names might get a little screwed up if `0.01` turns into something with 16 digits, so you might want to do some formatting magic there.

Reply:
I feel like I have a slight preference for sticking with the same set of nu values as in the paper, so the 4 values in my suggestion above.   That would make for an easier comparison with the model results from the paper if anyone wanted to do that. Maybe no one will actually want to do that and it's a moot point.

@mark-petersen, do you have an opinion?